client-libraries/python/auxon-sdk/src/ingest.rs

```diff
@@ -0,0 +1,92 @@
+use crate::{types, SdkError};
+use auxon_sdk::{
+    api::{AttrVal, TimelineId},
+    ingest_client::IngestStatus,
+};
+use pyo3::prelude::*;
+
+#[pyclass]
+pub struct IngestClient {
+    rt: tokio::runtime::Runtime,
+    client: auxon_sdk::plugin_utils::ingest::Client,
+}
+
+impl IngestClient {
+    pub fn new(
+        rt: tokio::runtime::Runtime,
+        client: auxon_sdk::plugin_utils::ingest::Client,
+    ) -> Self {
+        Self { rt, client }
+    }
+}
+
+#[pymethods]
+impl IngestClient {
+    pub fn disable_auto_timestamp(&mut self) {
+        self.client.disable_auto_timestamp();
+    }
+
+    pub fn switch_timeline(&mut self, id: TimelineId) -> Result<(), SdkError> {
```

Comment:
Does it matter that this returns SdkError while the others return PyErr?

Reply:
No, pyo3 does conversion automatically